rest_framework_filters/filterset.py

```diff
@@ -70,6 +70,17 @@ def related_filters(self):
             ])
         return self._related_filters
 
+    @property
+    def expanded_filters(self):
+        if '_expanded_filters' not in self.__dict__:
+            self._expanded_filters = self.base_filters.copy()
+            for filter_name, f in self.related_filters.items():
+                del self._expanded_filters[filter_name]
+                for related_name, related_f in six.iteritems(f.filterset.expanded_filters):
```

Comment:
DRFF is Python 3+, so it's not necessary to use `six`. Also, `six` will be remove in Django 3.

Reply:
Same as above. `six` is currently used in the codebase but yes, I get that it can be updated.